sys/fuchsia/clock.txt

```diff
@@ -4,13 +4,17 @@
 include <zircon/syscalls.h>
```

Comment:
Nit: include link to corresponding fidl file, i.e. zircon/vdso/clock.fidl

Reply:
Good point. added.